benchmark/code/minimatch.js

```diff
@@ -1,12 +1,11 @@
 'use strict';
 
-
 var minimatch = require('minimatch').match;
 var multimatch = require('multimatch');
 
 module.exports = function (files, patterns, options) {
   if (Array.isArray(patterns)) {
-    return multimatch(files, patterns, options);
+    return multimatch.apply(null, arguments);
```

Comment:
that's in the benchmarks. I updated `benchmarked` to latest and just tried to make sure args were being passed correctly to multimatch